thoth/metrics_exporter/jobs/db.py

```diff
@@ -119,9 +119,16 @@ def set_script_head_revision(cls):
     @register_metric_job
     def set_table_head_revision(cls):
         """Set metric for indicating database revision exposed by alembic table."""
-        metrics.database_schema_revision_table.labels(
-            "metrics-exporter", cls.graph().get_table_alembic_version_head(), Configuration.DEPLOYMENT_NAME
-        ).set(1)
+        table_revision_head = None
+        try:
+            table_revision_head = cls.graph().get_table_alembic_version_head()
+        except Exception as e:
+            _LOGGER.warning("Database revision table could not be retrieved: %r", e)
```

Comment:
Maybe an exception should be reported (but I don't know the issue behind this):

```suggestion
            _LOGGER.exception("Database revision table could not be retrieved: %r", e)
```